components/dashboard/ActivityInputDialog.js

```diff
@@ -137,19 +170,15 @@ const FEELINGS = [
   'Overwhelmed',
 ];
 
-// random shuffle of Feeling types.
-FEELINGS.sort(() => {
-  return 0.5 - Math.random();
-});
-
 const activityFormValues = {
-  activityType: ACTIVITY_TYPES[0],
+  activityTypeValue: ACTIVITY_TYPES[0].value,
```

Comment:
Any reason that we are breaking apart the object here? I think we can send the`activityType` object to state which saves us from having to always keep this in sync (like in https://github.com/newjersey/career-network/pull/82/files#diff-b390faea8b51ebe227e05e3256187276R277-R282)

Reply:
Good call, definitely cleaner to just pass the object to the select component. The only thing is that I think I would like to store them in Firestore as individual values rather than a nested object, for facilitation of querying down the line (we get automatic indexing on "top level" object properties but not on properties of nested objects).

I would say let's just store the slug -- I see that as the authoritative piece of data. I just want to record the label also so we know exactly what language the user saw when they selected the value. Although, we could also get that from the config object we shove into the record too, but that makes it a little more complex to retrieve when displaying in a log (and I'm really leaning into the whole no-SQL denormalization optimize-for-reads thing here).

But agreed that my code here it kind of ugly.

What do you think?

Reply:
Let's keep it for now. 

Agreed about the representation in Firestore (and thanks for the reminder about the auto-indexing! What a great feature.)